app/models/webcast/merged.rb

```diff
@@ -14,79 +14,21 @@ def initialize(uid, course_policy, term_yr, term_cd, ccn_list, options = {})
     def get_feed_internal
       logger.warn "Webcast merged feed where year=#{@term_yr}, term=#{@term_cd}, ccn_list=#{@ccn_list.to_s}, course_policy=#{@course_policy.to_s}"
       @academics = MyAcademics::Teaching.new(@uid)
-      feed = { :system_status => Webcast::SystemStatus.new(@options).get }
-      feed.merge get_media_feed
+      get_media_feed
     end
 
     private
 
     def get_media_feed
       media = get_media
-      feed = get_eligible_for_sign_up media
       if media.any?
         media_hash = {
           :media => media,
           :videos => merge(media, :videos)
         }
-        feed.merge! media_hash
-      end
-      feed
-    end
-
-    def get_eligible_for_sign_up(media)
-      eligible_for_sign_up = []
-      can_sign_up_one_or_more = false
-      is_sign_up_active = Webcast::SystemStatus.new(@options).get[:isSignUpActive]
-      can_view_webcast_sign_up = @course_policy.can_view_webcast_sign_up?
-      logger.warn "Course policy for user #{@uid} says can_view_webcast_sign_up = #{can_view_webcast_sign_up}"
-      if is_sign_up_active && @term_yr && @term_cd && can_view_webcast_sign_up
```

Comment:
For history, the call to `can_view_webcast_sign_up` has been returning `false` for years, because the feature flag (`webcast_sign_up_on_calcentral`) was never set as active. So all this code for this "sign up" feature doesn't run. The result has always been the following hash with media merged into it.

```ruby
{
  :userCanSignUpOneOrMore => false,
  :eligibleForSignUp => []
}
```